engine/src/api/config/settings.py

```
@@ -43,25 +43,24 @@ class Settings(BaseSettings):
     # Workflow Settings
     WORKFLOW_EXECUTION_TIMEOUT: int = 300
 
-    # Logging Settings
-    LOG_LEVEL: str = "INFO"
+    LLM_MODEL: Optional[str] = Field(default="openai/gpt-4o", env="LLM_MODEL") # Format: provider/model_name
```

Ruff (0.11.9)

Line too long (108 > 100)

(E501)